internal/pkg/manifest/backend_svc.go

```diff
@@ -122,7 +122,8 @@ func (s BackendService) ApplyEnv(envName string) (*BackendService, error) {
 	// Apply overrides to the original service s.
 	err := mergo.Merge(&s, BackendService{
 		BackendServiceConfig: *overrideConfig,
-	}, mergo.WithOverride, mergo.WithOverwriteWithEmptyValue)
+	}, mergo.WithOverride, mergo.WithOverwriteWithEmptyValue, mergo.WithTransformers(imageTransformer{}))
```

Comment:
We might want to give `imageTransformer` another name since it's not actually a transformer for image, it's just for now we only customize the merging of image type. Maybe `workloadTransformer`?

Reply:
addressed!